assets/src/scripts/services/statistics.js

```diff
@@ -1,8 +1,8 @@
 import { post } from '../lib/ajax';
 
 // median water level URL
-//const MWL_URL = `${config.SERVICE_ROOT}/statistics/calculate`;
-const MWL_URL = 'http://localhost:8777/statistics/calculate';
+const MWL_URL = `${config.SERVICE_ROOT}/statistics/calculate`;
```

Comment:
Should remove commented out code. Also you might want to consider creating a new config.py variable for the statistics service root. As I understand it, this is a separate tomcat instance from the rest of the NGWMN services.

Reply:
I will do this when I update it for the new mappings that Shirley is working one.

Reply:
If you do that you will need to update the ansible playbook which deploy the app and config: https://code.chs.usgs.gov/wma/ngwmn/ngwmn_ui_config

Reply:
Thanks for the heads up.